### PR TITLE
Add Docker samples

### PR DESCRIPTION
**Pull request by dermeister0**
Docker configuration and tasks for sample projects.

### CODE REVIEW

File: scripts/SaritasaRulesetInstall.ps1 [hunk 1]
```diff
@@ -1,45 +0,0 @@
-#
```
Comment (krasninja) [inline scripts/SaritasaRulesetInstall.ps1:1]: Why need to remove this? Let's have the option to use it both global and local

**Comment by krasninja** [comment]
Pls update samples/README.md with info how to use Docker (what to install, commands) and how to switch db provider for ZergRush project.

File: default.ps1 [hunk 2]
```diff
@@ -1,21 +1,26 @@
...
 
 . .\scripts\BuildTasks.ps1
+. .\scripts\DockerTasks.ps1
 . .\scripts\PublishTasks.ps1
 
-Import-Module Saritasa.Build
-Import-Module Saritasa.Test
+Properties `
+{
+    $Version = '0.1.0'
```
Comment (krasninja) [inline default.ps1:20]: What does the version stand for?

File: samples/Saritasa.BoringWarehouse/Docker/Dockerfile.web [hunk 3]
```diff
@@ -0,0 +1,16 @@
+FROM microsoft/aspnet
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop';"]
+
+RUN Set-ItemProperty -Path 'HKLM:\SYSTEM\CurrentControlSet\Services\Dnscache\Parameters' -Name ServerPriorityTimeLimit -Value 0 -Type DWord
```
Comment (krasninja) [inline samples/Saritasa.BoringWarehouse/Docker/Dockerfile.web:4]: Pls put comment why is it needed

File: samples/ZergRushCo.Todosya/ZergRushCo.Todosya.Web/Web.config [hunk 4]
```diff
@@ -5,7 +5,7 @@
     <section name="nlog" type="NLog.Config.ConfigSectionHandler, NLog" />
   </configSections>
   <connectionStrings>
-    <add name="AppDbContext" connectionString="Data Source=|DataDirectory|\ZergRushCo.Todosya.sqlite" providerName="System.Data.SQLite" />
+    <add name="AppDbContext" connectionString="Data Source=172.22.245.62;Initial Catalog=ZergRushCo;User ID=TestUser;Password=gHJT2SCm" providerName="System.Data.SqlClient" />
```
Comment (krasninja) [inline samples/ZergRushCo.Todosya/ZergRushCo.Todosya.Web/Web.config]: Avoid having ip address hardcoded here. Pls add commented connection string how Sqlite can be used instead.

File: default.ps1 (hunk 2 above)
Comment (dermeister0) [reply default.ps1:20]: It's used in CompileDocs function.

    (Get-Content "$docsRoot\conf.py").Replace('VX.VY', $Version) | Set-Content "$docsRoot\conf.py"

File: samples/ZergRushCo.Todosya/ZergRushCo.Todosya.Web/Web.config (hunk 4 above)
Comment (dermeister0) [reply samples/ZergRushCo.Todosya/ZergRushCo.Todosya.Web/Web.config]: Reverted.

File: scripts/SaritasaRulesetInstall.ps1 (hunk 1 above)
Comment (dermeister0) [reply scripts/SaritasaRulesetInstall.ps1:1]: Restored script.

**Pull request merged**